Makefile

```diff
@@ -48,7 +48,7 @@ dev:
 	@if docker ps -a --filter name=apisonator-dev | grep -q $(IMAGE_NAME) 2> /dev/null >&2; then \
 		docker start -ai apisonator-dev ; \
 	else \
-		docker run -ti -h apisonator-dev -v \
+		docker run -ti -h apisonator-dev --expose=3000 -p 3000:3000 -v \
```

Comment:
BTW it is pretty common to be able to configure local port by `PORT` env variable. 
That would be helpful for people running more than one ruby app locally.

Reply:
I understand you want to be able to modify the port in use in the local host machine so it does not cause conflict with other local ports of other service that may use the same port.

I performed a new commit that would allow this with an environment variable.

Reply:
Yes. And common name for this variable is `PORT`.

Reply:
Hi,

I agree that usually PORT name is a good name but I chose the name BACKEND_SERVICE_PORT in order to be able to differentiate it from other potentially configurable ports in this Makefile for other services/actions.
For example, if in the future we want to be able to modify the container port we would need another environment variable, which would have to have another name. The same think would happen if we want to be able to control other ports on other services.

Reply:
If this command starts one service that starts listening on one port then I really don't see the need for namespaced name. If there would be need in the future to customize other ports, then those could be named differently, because PORT is the port the service starts listening on.
I don't really like making compromises now for future that might never arrive.

Good to merge anyway 👍 at least it is configurable (for someone who is going to check the makefile).

Reply:
Hi mikz,

Seems reasonable what you are saying. I updated the variable name and added documentation on the Makefile so people can be aware of this variable.